independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PaySpace",
+    "description": "Our powerful API allows you to access your employee data in order to utilize in your business environment. All supported endpoints in one place and ready to be integrated into your solution.",
```

Comment:
@mint-virgil Please remove the work "API" from the description. please follow our requirements [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
@Amjed-Ayoub noted on this item